modules/cluster-logging-loki-about.adoc

```diff
@@ -0,0 +1,46 @@
+// Module is included in the following assemblies:
+//cluster-logging-loki.adoc
+:_content-type: CONCEPT
+[id="about-logging-loki_{context}"]
+= About the LokiStack
+
+In {logging} documentation, *LokiStack* refers to the {logging} supported combination of Loki, and web proxy with {product-title} authentication integration. LokiStack's proxy uses {product-title} authentication to enforce multi-tenancy. *Loki* refers to the log store as either the individual component or an external store.
+
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system currently offered as an alternative to Elasticsearch as a log store for the {logging}. Elasticsearch indexes incoming log records completely during ingestion. Loki only indexes a few fixed labels during ingestion, and defers more complex parsing until after the logs have been stored. This means Loki can collect logs more quickly. As with Elasticsearch, you can query Loki link:https://grafana.com/docs/loki/latest/[using JSON paths or regular expressions].
+
+[id="deployment-sizing_{context}"]
+== Deployment Sizing
+Sizing for Loki follows the format of `N<x>._<size>_` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
+
+[NOTE]
+====
+1x.extra-small is for demo purposes only, and is not supported. 
+====
+
+.Loki Sizing
+[options="header"]
+|========================================================================================
+|                              | 1x.extra-small  | 1x.small            | 1x.medium
+| *Data transfer*              | Demo use only.  | 500GB/day           | 2TB/day
+| *Queries per second (QPS)*   | Demo use only.  | 25-50 QPS at 200ms  | 25-75 QPS at 200ms
+| *Replication factor*         | None            | 2                   | 3
+| *Total CPU requests*         | 5 vCPUs         | 36 vCPUs            | 54 vCPUs
+| *Total Memory requests*      | 7.5Gi           | 63Gi                | 139Gi
+| *Total Disk requests*        | 150Gi           | 300Gi               | 450Gi
+|========================================================================================
+
+[id="CRD-API-support_{context}"]
+== Supported API Custom Resource Definitions
+LokiStack development is ongoing, not all APIs are supported currently supported.
+
+[options="header"]
+|=====================================================================
+| CustomResourceDefinition (CRD)| ApiVersion           | Support state
+| LokiStack      | lokistack.loki.grafana.com/v1       | Supported in 5.5
```

Comment:
Logging 5.5 is not supported in Openshift 4.8.  Logging 5.4 is the largest version in Open-shift 4.8,  Loki operator is tech-preview in Logging 5.4.